src/hubverse_transform/model_output.py

```diff
@@ -0,0 +1,152 @@
+import logging
+import pathlib
+import re
+
+import pyarrow as pa
+import pyarrow.parquet as pq
+from pyarrow import csv, fs
+
+# Log to stdout
+logger = logging.getLogger(__name__)
+handler = logging.StreamHandler()
+formatter = logging.Formatter('%(asctime)s -  %(levelname)s - %(name)s - %(message)s', datefmt='%m/%d/%Y %I:%M:%S %p')
+handler.setFormatter(formatter)
+logger.addHandler(handler)
+logger.setLevel(logging.INFO)
+
+
+class ModelOutputHandler:
+    def __init__(self, input_uri: str, output_uri: str):
+        input_filesystem = fs.FileSystem.from_uri(input_uri)
+        self.fs_input = input_filesystem[0]
+        self.input_file = input_filesystem[1]
+
+        output_filesystem = fs.FileSystem.from_uri(output_uri)
+        self.fs_output = output_filesystem[0]
+        self.output_path = output_filesystem[1]
+
+        # get file name and type from input file
+        path = pathlib.Path(self.input_file)
+        self.file_name = path.stem
+        self.file_type = path.suffix
+
+        # TODO: Add other input file types as needed
+        if self.file_type not in ['.csv', '.parquet']:
```

Comment:
In future iterations, it wouldn't be difficult to add other file formats here as needed.

Reply:
Currently we definitely support '.arrow' so might as well add it.
https://github.com/Infectious-Disease-Modeling-Hubs/schemas/blob/558591226b4bd93f75a6e85158395d17df584c20/v2.0.1/admin-schema.json#L97-L100